internal/executor/doctor/executor.go

```diff
@@ -185,13 +184,16 @@ func (d *Executor) getGptClient(cfg *Config) (gpt3.Client, error) {
 		return nil, fmt.Errorf("OpenAPI API Key cannot be empty. You generate it here: https://platform.openai.com/account/api-keys")
 	}
 
-	var opts []gpt3.ClientOption
-	if cfg.APIBaseURL != "" {
-		opts = append(opts, gpt3.WithBaseURL(cfg.APIBaseURL))
+	defaultEngine := gpt3.TextDavinci003Engine
```

Comment:
Maybe we should do the same for other properties? Because described defaults are assumptions based on the underlying implementation. While bumping deps, it may change so OS and cloud will work differently as OS won't have those taken from JSON Schema.